### DISCUSSION TITLE
HM-400 not detected (NRF24)

### DISCUSSION THREAD

**Discussion by fouille** [tbnobody/OpenDTU#1203] [Q&A] [answered]
Hello, 
I have another problem. 
now that my NRF24 card is correctly detected, my HM-400 is not. I think I tried lots of solutions, closer to the inverters (about 2m), restarting, changing the range (to max). nothing helps it does not find my inverter (Sunology Play - HM-400 inverter).

![IMG_2001](https://github.com/tbnobody/OpenDTU/assets/24554441/f2152790-3461-493d-9e7d-fbb56c931463)
![IMG_1999](https://github.com/tbnobody/OpenDTU/assets/24554441/678b8c4b-c478-443f-abb1-bd77a88c5992)
![IMG_1998](https://github.com/tbnobody/OpenDTU/assets/24554441/ec190f88-377f-490a-993b-046fda4c2cc2)

can you help me? what do I need to give you?

**Comment by tbnobody** [comment]
If the Web UI shows a valid connection for the NRF module it is still possible that the IRQ pin is not properly connected. 
Please also double check the serial number of your inverter.

**Comment by stefan123t** [comment]
@fouille please also check the output under Info > Console to that end (IRQ).

There should be messages (Interrupt received / Success) that tell you it is working fine and you receive packets:
```
15:19:33.350 > Fetch inverter: 114112345678
15:19:33.395 > TX RealTimeRunData Channel: 75 --> 15 12 34 56 78 87 65 43 21 80 0B 00 64 CB A9 65 00 00 00 00 00 00 00 00 5D 48 08 
15:19:33.420 > Interrupt received
15:19:33.425 > RX Channel: 40 --> 95 12 34 56 78 12 34 56 78 01 00 01 01 46 03 13 0A 02 01 5B 01 E2 06 85 00 07 F7 | -80 dBm
15:19:33.460 > Interrupt received
15:19:33.475 > RX Channel: 23 --> 95 12 34 56 78 12 34 56 78 02 68 1B 00 07 23 2B 05 53 04 5C 09 39 13 88 0F C9 88 | -80 dBm
15:19:33.498 > Interrupt received
15:19:33.512 > RX Channel: 23 --> 95 12 34 56 78 12 34 56 78 83 00 00 00 AB 03 E8 01 82 00 03 BC C5 AF | -80 dBm
15:19:33.894 > RX Period End
15:19:33.946 > Success
```

**Comment by fouille** [comment, marked as answer]
Hi guys,

Thank you for your answers, I may have written a little too quickly, I'm sorry.
Without knowing how to explain it, it seems that it comes from my boiler located next to the electrical outlet that supplies my DTU.
When I switch off the boiler (EVACALOR) or when I go outside next to the panels it works perfectly.

I think the boiler emits a wave that disturbs the reception.
I will therefore better optimize the location of my DTU box.

With my apologies for the inconvenience.
Frank
